README.md

```diff
@@ -15,6 +15,31 @@ A state controller with its own debugger
 ## Please head over to our WIP website for cerebral v2
 [http://www.cerebraljs.com/](http://cerebral.github.io/cerebral-website). You will find all the information you need there.
 
+## Contribute
+The entire Cerebral codebase has been rewritten to encourage contributions. The code is cleaned up, commented and all code is in a "monorepo". That means you can run tests across projects and general management of the code is simplified a lot.
+
+1. Clone the monorepo
+2. In root: `npm install` (Do not use YARN)
```

Comment:
we can omit yarn mention here. nothing wrong in installing monorepo deps with yarn. but lerna will use npm while bootstrap anyway